sklearn/feature_selection/_from_model.py

```diff
@@ -174,9 +192,16 @@ class SelectFromModel(MetaEstimatorMixin, SelectorMixin, BaseEstimator):
     array([False,  True, False])
     >>> selector.transform(X)
     array([[-1.34],
-           [-0.02],
-           [-0.48],
-           [ 1.48]])
+          [-0.02],
+          [-0.48],
+          [ 1.48]])
```

Comment:
We can revert this too since it is unrelated to the PR.

Reply:
Done. Not sure how that happened 😅